apps/server/src/stores/runtimeState.ts

```diff
@@ -654,5 +650,33 @@ export function roll(rundown: OntimeRundown): { eventId: MaybeString; didStart:
 
   // update runtime
   runtimeState.runtime.actualStart = runtimeState.clock;
+  runtimeState.currentBlock.startedAt = runtimeState.clock;
   return { eventId: runtimeState.eventNow.id, didStart: true };
 }
+
+function loadBlock(
+  rundown: OntimeRundown,
+  prevBlockId: string | undefined = runtimeState.currentBlock.block?.id,
+  prevBlockStartAt: number | null = runtimeState.currentBlock.startedAt,
```

Comment:
question: I am not sure I understand this condition.

loadBlock has access to the state, why not consume it directly?